src/config.rs

```diff
@@ -68,6 +68,7 @@ pub struct CoreConfig {
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
 pub struct VaultConfig {
     pub api_token: String,
+    pub expiry_time: Option<String>,
```

Comment:
This is to support the backward compatibility, as some users already has the token without expiry

Is there any workaround for this?

Reply:
I see.

@onimsha since this is still pre 1.0, can we still introduce breaking change? Users have to run `wukong init` again to make it work.

Reply:
> I see.
> 
> @onimsha since this is still pre 1.0, can we still introduce breaking change? Users have to run `wukong init` again to make it work.

Yes, I guess we can ask them to do it one more time.